CMakeLists.txt

```diff
@@ -71,6 +71,26 @@ include(cmake/profile.cmake)
 include(cmake/module.cmake)
 include(cmake/thread.cmake)
 
+# Fuzzers are compiled without PIC support,
+# LuaJIT in FreeBSD doesn't work with PIC (gh-7640),
+# ligomp.a for AArch64 CentOS is compiled without PIC support.
+if (ENABLE_FUZZER OR TARGET_OS_FREEBSD OR ${CMAKE_SYSTEM_PROCESSOR} MATCHES "aarch64")
```

Comment:
Why hardening is disabled on arm64?

Reply:
Some arm64 Linux distros (e.g. CentOS) include system libraries that do not allow to compile Tarantool as position-independent binary, that's why I disabled hardening by default for arm64. There is a task about removing dependency on libgomp: #7689